src/patternfly/demos/Tabs/templates/tabs--table-overflow-menu.hbs

```diff
@@ -1,7 +1,15 @@
 {{#> overflow-menu dropdown-menu--modifier="pf-m-align-right" overflow-menu--id=tabs--table-overflow-menu--id}}
-  {{#> overflow-menu-control overflow-menu-button--aria-label="Dropdown for tabs table"}}
-    {{#> overflow-menu-dropdown-item}}
-      Action Link
-    {{/overflow-menu-dropdown-item}}
-  {{/overflow-menu-control}}
+  {{#if dropdown--IsExpanded}}
```

Comment:
This looks good. My question about moving the `{{#if/else}}` block in the `toolbar-overflow-menu-example.hbs` template is basically asking if we can update that template's conditional to just wrap `{{#> overflow-menu-control}}` as you've done here so we have less repeated code.